policy/release/rpm_repos.rego

```diff
@@ -35,16 +35,16 @@ deny contains result if {
 # description: >-
 #   Each RPM package listed in an SBOM must specify the repository id that it comes from,
 #   and that repository id must be present in the list of known and permitted repository ids.
+#   Currently this is rule enforced only for SBOM components created by cachi2.
 # custom:
 #   short_name: ids_known
 #   failure_msg: 'RPM repo id check failed: %s'
 #   solution: >-
 #     Ensure every rpm comes from a known and permitted repository, and that the data in the
 #     SBOM correctly records that.
-#   # Todo: Until the sbom generation is upated this will always fail, so don't include it
-#   # in the redhat collection yet. See https://issues.redhat.com/browse/STONEBLD-2638
-#   #collections:
-#   #- redhat
+#   collections:
+#   - redhat
+#   effective_on: "2024-11-10:00:00Z"
```

Comment:
I believe this is the right format: `2024-11-10T00:00:00Z`

(We should probably have a generic check for this :thinking:)

Reply:
#1183

Reply:
Dang did I leave out the seconds? Thanks.

Reply:
wait I left out a few things.. :sweat: